src/pages/Portfolio/MyKiva/MyKivaPage.vue

```diff
@@ -18,22 +26,43 @@ export default {
 	name: 'MyKivaPage',
 	inject: ['apollo', 'cookieStore'],
 	components: {
+		MyKivaNavigation,
 		WwwPage,
 		MyKivaHero,
 		MyKivaProfile
 	},
 	data() {
 		return {
 			lender: null,
+			showNavigation: false,
+			userInfo: {},
 		};
 	},
 	apollo: {
 		query: myKivaQuery,
 		preFetch: true,
 		result(result) {
+			this.userInfo = result.data?.my ?? {};
 			this.lender = result.data?.my?.lender ?? null;
 		},
 	},
+	computed: {
+		userBalance() {
+			return this.userInfo?.userAccount?.balance ?? '';
+		},
+		lenderName() {
+			return this?.lender?.name ?? '';
+		},
+		lenderImageUrl() {
+			return this?.lender?.image?.url ?? '';
+		},
+	},
+	methods: {
+		handleShowNavigation() {
+			this.showNavigation = true;
+			this.$kvTrackEvent('SecondaryNav top level', 'click', 'MyKiva-Settings-icon');
```

Comment:
We need to use pre-approved categories for events, so this should probably just have a category of `borrower-profile`

Reply:
Agreed, but this category is the one mentioned in the ticket: https://kiva.atlassian.net/browse/MP-729
cc @emuvente

Reply:
Yeah, just follow what's in the ticket and don't worry about whether the event details are following the web events guide. It does not seem to be something that product or analytics care about strongly any more.

Reply:
@emuvente I had thought we needed to only use validated strings, but I see that this list doesn't include categories that we are currently using, for example `Lending`, so it must not be a requirement.

https://github.com/kiva/analytics-notebooks/blob/446439ec4993aaf5f8e05c18dc7e674d8e72b509/notebooks/analytics/structured_event_validation/se_validation.json

Reply:
May want to check with Greg to be sure we can import these new event types into Amplitude if that is expected. I think that list is related to the automated export/import into that tool.

Reply:
Ah, so Looker will work with anything, but potentially not Amplitude.